examples/camera-app/linux/src/clusters/push-av-stream-transport/push-av-stream-manager.cpp

```diff
@@ -222,18 +238,96 @@ Protocols::InteractionModel::Status PushAvStreamTransportManager::ManuallyTrigge
     }
 
     ChipLogProgress(Camera, "PushAvStreamTransportManager, Trigger PushAV Transport for Connection: [%u]", connectionID);
+    if (timeControl.HasValue())
+    {
+        mTransportMap[connectionID]->ConfigureRecorderTimeSetting(timeControl.Value());
+    }
     mTransportMap[connectionID]->TriggerTransport(activationReason);
 
     return Status::Success;
 }
 
+void PushAvStreamTransportManager::GetBandwidthForStreams(const Optional<DataModel::Nullable<uint16_t>> & videoStreamId,
+                                                          const Optional<DataModel::Nullable<uint16_t>> & audioStreamId,
+                                                          double & outBandwidthMbps)
+{
+    outBandwidthMbps = 0.0;
+
+    if (videoStreamId.HasValue() && !videoStreamId.Value().IsNull())
+    {
+        uint16_t vStreamId = videoStreamId.Value().Value();
+
+        auto & availableVideoStreams = mCameraDevice->GetCameraHALInterface().GetAvailableVideoStreams();
```

Comment:
This needs to calculate against the allocated streams' bitrate configurations, not the available streams in the camera device.
So, towards that, this function should call into the CameraAVStreamMgmt delegate to calculat ethe stream bandwidth because it has access to the allocated streams.